Source/Core/InputCommon/ControllerInterface/evdev/evdev.cpp

```diff
@@ -184,6 +184,12 @@ void Init()
   StartHotplugThread();
 }
 
+void GetDevices()
```

Comment:
> IMO, GetDevices is not the correct name for what this does.

I agree with you, but right now I'm blanking on a better name. Any ideas?

> Also, you probably don't need to shut down completely, i.e. it is unnecessary to shut down the hotplug thread and start it again simply for re-adding devices. I suggest leaving the hotplug stuff in Init and Shutdown, and moving the device enumerating part to this function.

Done.